tools/arriba.wdl

```diff
@@ -0,0 +1,325 @@
+
+    command <<<
+        arriba \
+            -x ~{bam} \
+            ~{if defined(chimeric_sam) then "-c " + chimeric_sam else ""} \
+            -o ~{prefix}.tsv \
+            -O ~{prefix}.discarded.tsv \
+            -a ~{reference_fasta_gz} \
+            -g ~{gtf} \
+            -G "~{feature_name}" \
+            ~{if defined(exclude_list) then "-b " + exclude_list else ""} \
+            ~{if defined(known_fusions) then "-k " + known_fusions else ""} \
+            ~{if defined(annotate_fusions) then "-t " + annotate_fusions else ""} \
+            ~{if defined(protein_domains) then "-p " + protein_domains else ""} \
+            ~{if defined(wgs_svs) then "-d " + wgs_svs else ""} \
+            -D ~{max_genomic_breakpoint_distance} \
+            -s ~{strandedness} \
+            ~{if length(interesting_contigs) > 0 then "-i " + sep(',', interesting_contigs) else ""} \
+            ~{if length(viral_contigs) > 0 then "-v " + sep(',', viral_contigs) else ""} \
+            ~{if length(disable_filters) > 0 then "-f " + sep(',', disable_filters) else ""} \
+            -E ~{max_e_value} \
+            -S ~{min_supporting_reads} \
+            -m ~{max_mismappers} \
+            -L ~{max_homolog_identity} \
+            -H ~{homopolymer_length} \
+            -R ~{read_through_distance} \
+            -A ~{min_anchor_length} \
+            -M ~{many_spliced_events} \
+            -K ~{max_kmer_content} \
+            -V ~{max_mismatch_pvalue} \
+            -F ~{fragment_length} \
+            -U ~{max_reads} \
+            -Q ~{quantile} \
+            -e ~{exonic_fraction} \
+            -T ~{top_n} \
+            -C ~{coverage_fraction} \
+            -l ~{max_itd_length} \
+            -z ~{min_itd_allele_fraction} \
+            -Z ~{min_itd_supporting_reads} \
+            ~{if mark_duplicates then "" else "-u"} \
+            ~{if report_additional_columns_discard then "-X" else ""} \
+            ~{if fill_gaps then "-I" else ""}
+    >>>
+
+    output {
+        File fusions = "~{prefix}.tsv"
+        File discarded_fusions = "~{prefix}.discarded.tsv"
+    }
+
+    runtime {
+        cpu: 1
+        memory: "~{memory_gb} GB"
+        disks: "~{disk_size_gb} GB"
+        container: "quay.io/biocontainers/arriba:2.4.0--h0033a41_2"
+        maxRetries: 1
+    }
+}
+
+task arriba_tsv_to_vcf {
+    meta {
+        description: "Convert Arriba TSV format fusions to VCF format."
+        outputs: {
+            fusions_vcf: "Output file of fusions in VCF format"
+        }
+    }
+
+    parameter_meta {
+        fusions: "Input fusions in TSV format to convert to VCF"
+        reference_fasta_gz: "Gzipped reference genome in FASTA format"
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File fusions
+        File reference_fasta_gz
+        String vcf = basename(fusions, ".tsv") + ".vcf"
```

Comment:
This is fine, if you understand you're letting users create VCF files with non-`.vcf` extensions. I would guess you'd rather enforce the .vcf filename?
```suggestion
        String prefix = basename(fusions, ".tsv")
```